build/media-suite_compile.sh

```diff
@@ -1970,31 +1970,38 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
     fi
 
     _check=(libvulkan.a vulkan.pc)
-    if ! mpv_disabled vulkan &&
-        do_vcs "https://github.com/KhronosGroup/Vulkan-Loader.git#tag=v1.1.114" vulkan-loader; then
-        _shinchiro_patches="https://raw.githubusercontent.com/shinchiro/mpv-winbuild-cmake/b1f8bc13/packages"
-        do_uninstall "${_check[@]}"
-        do_patch "$_shinchiro_patches/vulkan-0001-cross-compile-static-linking-hacks.patch"
-        create_build_dir
-        log dependencies /usr/bin/python3 ../scripts/update_deps.py --no-build
-        cd_safe Vulkan-Headers
-            _check=(vulkan/vulkan.h)
-            do_uninstall include/vulkan
-            do_cmakeinstall
+    if ! mpv_disabled vulkan; then
+
+        _check=(d3dkmthk.h d3dukmdt.h)
+        if ! files_exist "${_check[@]}" && do_vcs "https://github.com/DeadSix27/python_cross_compile_script.git"; then
+            do_uninstall "${_check[@]}"
+            do_install additional_headers/d3d{kmthk,ukmdt}.h include/
```

Comment:
Aren't these static? Why are we adding a VCS for this?

Reply:
Now using curl to download the headers, is that OK?

Reply:
Use do_wget instead. See other cases where it's just downloading files in the scripts.